### ISSUE TITLE
[IOS] Library not loaded libswiftAVFoundation.dylib

### ISSUE THREAD

**Issue by davidgv88** [imgly/pesdk-cordova-demo#8]
Hi!

I have this error when starting the application.

<img width="1019" alt="captura de pantalla 2018-06-20 a las 13 28 37" src="https://user-images.githubusercontent.com/679262/41656196-737d6d7c-748f-11e8-9975-5e3fc45fefac.png">

The framework list:

<img width="733" alt="captura de pantalla 2018-06-20 a las 13 44 24" src="https://user-images.githubusercontent.com/679262/41656407-1b3b315c-7490-11e8-8c57-cc86983c74f9.png">

Steps for reproduce the error:

```
git clone https://github.com/imgly/pesdk-cordova-demo.git
```
(Put PhotoEditorSDK.framework file to pesdk-cordova-demo/src/ios/)

```
cordova create pesdk_test
cd pesdk_test
cordova plugin add ../pesdk-cordova-demo/
cordova platform add ios
cordova build ios
```

Open pesdk_test/platforms/ios/HelloCordova.xcodeproj

And click "Run"

**Comment by davidgv88** [comment]
In Build Settings the option Always Embed Swift Standard Libraries is "Yes"

**Comment by davidgv88** [comment]
Fix! I updated Xcode from 9.2 to 9.4.1
